packages/indexer/src/core/indexer.ts

```diff
@@ -263,20 +267,19 @@ export class Indexer {
 
     const [block, error] = await retry(
       async () => {
-        return await this.limit(async () => {
-          this.orchestrator.metrics.incRpcRequests(
-            RpcRequestMethod.GetBlock,
-            this.network,
-          );
-          const start = new Date().valueOf();
-          const r = await this.provider.getBlock(blockNumber);
-          this.orchestrator.metrics.observeRpcLatency(
-            RpcRequestMethod.GetBlock,
-            this.network,
-            new Date().valueOf() - start,
-          );
-          return r;
-        });
+        this.orchestrator.metrics.incRpcRequests(
```

Comment:
Love that you added latency metrics 😻 

Might be able to make this block into a function that takes in arguments, just a thought.

Reply:
absolutely. but lets do later, priority now is to publish it as a stable thing asap